test/lib/actions/ConfigAction.test.ts

```diff
@@ -9,7 +9,8 @@ import {CodeAnalyzerConfigFactory} from "../../../src/lib/factories/CodeAnalyzer
 import {EnginePluginsFactory} from '../../../src/lib/factories/EnginePluginsFactory';
 import {ConfigAction, ConfigDependencies, ConfigInput} from '../../../src/lib/actions/ConfigAction';
 import {AnnotatedConfigModel} from '../../../src/lib/models/ConfigModel';
-import {ConfigStyledYamlViewer} from '../../../lib/lib/viewers/ConfigViewer';
+import {ConfigStyledYamlViewer} from '../../../src/lib/viewers/ConfigViewer';
+import {ConfigActionSummaryViewer} from '../../../src/lib/viewers/ActionSummaryViewer';
```

Comment:
Testing through `ConfigAction` using the actual implementation, instead of using an Interface/Stub setup.